security/libu2f-server/files/patch-core-macro-fix.diff

```diff
@@ -0,0 +1,16 @@
+--- u2f-server/core.c.orig	2017-12-18 03:52:12.000000000 -0800
++++ u2f-server/core.c	2021-08-12 19:39:34.000000000 -0700
+@@ -44,6 +44,13 @@
+ #define u2fs_json_object_object_get(obj, key, value) (value = json_object_object_get(obj, key)) == NULL ? (json_bool)FALSE : (json_bool)TRUE
+ #endif
+ 
++#ifndef FALSE
++#define FALSE 0
++#endif
++#ifndef TRUE
++#define TRUE  1
++#endif
```

Comment:
Could you use the upstream patch and cite the URL? https://github.com/Yubico/libu2f-server/commit/f7c4983b31909299c47bf9b2627c84b6bfe225de

Reply:
I have modified the Portfile accordingly. If there is anything else, please let me know. Thank you.